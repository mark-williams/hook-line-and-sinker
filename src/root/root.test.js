import React from 'react';
import { shallow } from 'enzyme';
import Root from './root';

describe('root', () => {
  describe('snapshot', () => {
    it('renders', () => {
      const wrapper = shallow(<Root />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
