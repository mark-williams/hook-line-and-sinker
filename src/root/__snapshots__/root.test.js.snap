// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root snapshot renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Root />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <BrowserRouter>
        <Switch>
          <Route
            component={[Function]}
            path="/counter"
          />
          <Route
            component={[Function]}
            path="/simple-form"
          />
          <Route
            component={[Function]}
            path="/complex-form"
          />
          <Route
            component={[Function]}
            path="/data-retrieval"
          />
          <Route
            component={[Function]}
          />
        </Switch>
      </BrowserRouter>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Switch>
          <Route
            component={[Function]}
            path="/counter"
          />
          <Route
            component={[Function]}
            path="/simple-form"
          />
          <Route
            component={[Function]}
            path="/complex-form"
          />
          <Route
            component={[Function]}
            path="/data-retrieval"
          />
          <Route
            component={[Function]}
          />
        </Switch>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Route
              component={[Function]}
              path="/counter"
            />,
            <Route
              component={[Function]}
              path="/simple-form"
            />,
            <Route
              component={[Function]}
              path="/complex-form"
            />,
            <Route
              component={[Function]}
              path="/data-retrieval"
            />,
            <Route
              component={[Function]}
            />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/counter",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/simple-form",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/complex-form",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/data-retrieval",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <BrowserRouter>
          <Switch>
            <Route
              component={[Function]}
              path="/counter"
            />
            <Route
              component={[Function]}
              path="/simple-form"
            />
            <Route
              component={[Function]}
              path="/complex-form"
            />
            <Route
              component={[Function]}
              path="/data-retrieval"
            />
            <Route
              component={[Function]}
            />
          </Switch>
        </BrowserRouter>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Switch>
            <Route
              component={[Function]}
              path="/counter"
            />
            <Route
              component={[Function]}
              path="/simple-form"
            />
            <Route
              component={[Function]}
              path="/complex-form"
            />
            <Route
              component={[Function]}
              path="/data-retrieval"
            />
            <Route
              component={[Function]}
            />
          </Switch>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Route
                component={[Function]}
                path="/counter"
              />,
              <Route
                component={[Function]}
                path="/simple-form"
              />,
              <Route
                component={[Function]}
                path="/complex-form"
              />,
              <Route
                component={[Function]}
                path="/data-retrieval"
              />,
              <Route
                component={[Function]}
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/counter",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/simple-form",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/complex-form",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/data-retrieval",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
